srcpkgs/timeshift/template

```diff
@@ -1,20 +1,19 @@
 # Template file for 'timeshift'
 pkgname=timeshift
-version=22.06.1
+version=22.06.6
 revision=1
 build_style=gnu-makefile
 hostmakedepends="gettext pkg-config vala which"
 makedepends="libgee08-devel json-glib-devel gtk+3-devel vte3-devel libgirepository-devel"
-depends="rsync psmisc"
+depends="rsync psmisc cronie"
```

Comment:
May I inquire as to why `cronie` is included here?

As far as I can tell from seeing this template, it is up to the end user to choose his/her crontab daemon.

Reply:
Cronie was listed as optional dependency and I thought it would be good to include it. It seems it was not a good idea. I'm going to remove it.

Reply:
Humm... just realized the whole project moved to linuxmint github. And there is already an even newer version.